src/site/content/en/blog/native-file-system/index.md

```diff
@@ -216,17 +255,18 @@ takes a string, which is what we want for a text editor. But it can also take a
 [BufferSource][buffersource], or a [Blob][blob]. Finally, finish writing by
 calling `FileSystemWriter.close()`.
 
-{% Aside 'caution' %}
-  There's no guarantee that the contents are written to disk until
-  the `close()` method is called.
-{% endAside %}
+{% endDetails %}
 
-### What else is possible?
+### Storing file handles in IndexedDB
 
-Beyond reading and writing files, the Native File System API provides
-several other new capabilities.
+Starting in Chrome 82, you can save a file handle to IndexedDB. This makes
+it possible to keep a list of recently opened or edited files. For example
+you could offer to re-open the last file when the app is opened, or show a
+list of recently edited files within your app. You can also `postMessage`
+file handles between the same top-level origin, for example between tabs or
+from tabs to workers.
 
-#### Open a directory and enumerate its contents
+### Open a directory and enumerate its contents
 
 To enumerate all files in a directory, call `chooseFileSystemEntries()`
 with the `type` option set to `'openDirectory'`. The user selects a directory
```

Comment:
`'openDirectory'` is now `'open-directory'`

Reply:
D'oh, thought I caught all of these, thanks!